### DISCUSSION TITLE
Is the original file's metadata included in the return?

### DISCUSSION THREAD

**Discussion by miscellaneousmatter** [ReagentX/imessage-exporter#285] [Q&A] [answered]
Additionally, any general info about metadata would be helpful. Thank you!

**Comment by ReagentX** [comment, marked as answer]
Whatever metadata is stored on attachments when they are received is preserved.

When `imessage-exporter` is asked to copy attachments, it updates the last modified date and last access date to when the message was received, as both are set to the time of the copy operation performed during the export otherwise.

https://github.com/ReagentX/imessage-exporter/blob/c99043b5de373828322b95119223ed3fcdeee7a0/imessage-exporter/src/app/attachment_manager.rs#L87-L96
